### ISSUE TITLE
No Json object could be decoded

### ISSUE THREAD

**Issue by rassalas** [Kiv/poclbm#71]
I'm new to the GUI miner,   and am having difficulty.  When I try to start I see the error message no json object could be detected.  Any hints on what I ma be doing wrong?

**Comment by Kiv** [comment]
Hi, the error is pretty vague - can you post the exact message, as well as what OS, pool, and GPU you're using?

**Comment by rassalas** [comment]
I did resolve it,  Didn’t quite have my slush pool set up.

**Comment by rassalas** [comment]
Another problem I’m having 

It seems to be running ok with a hash rate of 86 Khas/s on the slush pool,  however no shares yet,  The strange thing I’m seeing is if I try to minimize the app

It goes away and disappears from task bar.    Windows 7

BC

**Comment by Kiv** [comment]
Is it still running in Task Manager? Or has it exited?

**Comment by rassalas** [comment]
It appears not,  unless it’s called something other than guiminer in the task manager.

**Comment by Kiv** [comment]
Hm, really strange. It should be called guiminer.exe. Is there anything in
a guiminer.log file in the same directory as the EXE?

**Comment by rassalas** [comment]
Yes  the log file is there it has a lot of jumbled stuff and then at the bottom the time I started it today. 

One thing I’ve done that might not be good is created a shortcut and put that on my startmenu, I’ve been clicking that to start it.

I’m now starting to think that may be a problem.  I may restart not using the shortcut but directly with the .exe

And see if that makes a difference..

**Comment by rassalas** [comment]
So I’m using the application from it’s folder as opposed to a shortcut on my start menu, and it makes no difference.  It behaves the same.

A little more details on the behavior.  When I start the application it shows up in my task bar.  Even though start minimized is unchecked.  

If I want to see what it’s doing  and I go down to the tray and right click the guiminer icon I see only guiminer, pin this program to task bar, and close

If I click the guiminer option it starts a second instance of the program.

  If instead I hover over it a smaller version of the program comes up, if I right click that, a menu come up that says restore,move, size, ninimize, maximize and close, with the restore option being greyed out.  If I select maximize I can bring the program up to view,   when It’s open as I mentioned the minimize button makes it disappear,  the restore down minimizes it to the taskbar if I instead grab the top of the appl and drag it snaps to about the right size.   

  The X closes it.  

  One other thing in all this time I’ve been tinkering I’ve gotten hash rates as high as ~ 1.2 Mhash/ second  but have yet to see one share accepted.

Also If I restart the computer I am asked to save multiple times I guess for each time I’ve lost the handle on it.  

Also if I click the x and close it, then restart, I get about half the hash rate as before.

That’s a pretty complete summary of what I’m seeing as far as I  can tell.

Brendan

**Comment by Kiv** [comment]
Hi Brendan, I'm sorry to say I have no idea what your issue could be. Windows 7 should be well supported by GUIMiner. All I can suggest for you is to post what you've shown me on the Bitcoin forums just in case someone has experienced a similar issue. It's a pretty friendly place there, so someone might be able to help you.

**Comment by techman05** [comment]
last time I saw "No Json object could be decoded" it seemed like it was the poclbm.ini file where something had messed up. I had this the first few days till I saw on the bitcointalk.com forum that it was a bad version of the file. I stopped all mining if any,deleted the file while guiMiner was up and saved again and started mining again. Its not pasting all the opens and closes fields on overwriting.

That's my thought. I hope that helps you @Kiv in your bug squashing

**Comment by t0mmyBerg** [comment]
I have been having this problem now for a few days.  I had been mining slush's pool just fine for weeks.  computer was off for a few days and when i brought it back up it no longer connects.  No JSON Object could be decoded.  Then IO errors- 1, tolerance 2.  Windows 7 64 bit.  Running from the folder where it has been extracted.  Downloaded GUIMiner again and put it somewhere else.  Stopped the running instance and then ran it from the new location and it came up with the same login information as if I had invoked it from the other location.  I was expecting to have to re enter my slushs pool login information.  So the initilization is reading from somewhere external to the extraction directory, or it is reading it from memory I suppose.  But wheh I checked what was running in task manager I did not see anything.  Any ideas why I am not able to connect?

**Comment by rassalas** [comment]
When I had that problem it turned out to be just my login at the pool.  Once I straightened it that went away.  What pool are you using?  Slush pool changed the preferred pool address to the Stratum server  you might have to update the addresses and save them.

**Comment by t0mmyBerg** [comment]
Fails using stratum address as well.  if i put stratum.bitcoin.cz as host and change port to 3333 i get "checking for stratum"  then "no response to get work, using as stratum"  then "Failed to subscribe" then  "IO errors - 2, tolerance 2".  Like I said it was working fine last week, now not so much.

**Comment by misjudgedinall** [comment]
I am having the exact same issue. And I think my information may add more that point to an issue with the pool itself. But I'd like someone else to tell me what they think.

First off I use three computers to mine. Secondly I had my power go out for a day and the issue didn't begin until I attempted to start mining again the day after. Other than that no change. And again the issue occurred simultaneously on all three of my computers. 

I use slush's pool
I have GUI miner
I don't do anything special to set it up

Error is as follow pasted form the guiminer log file;

2013-06-06 22:43:08: Listener for "Default" started
2013-06-06 22:43:08: Listener for "Default": 06/06/2013 22:43:08, started OpenCL miner on platform 0, device 0 (Pitcairn)
2013-06-06 22:43:08: Listener for "Default": api2.bitcoin.cz:8332 06/06/2013 22:43:08, checking for stratum...
2013-06-06 22:43:09: Listener for "Default": api2.bitcoin.cz:8332 06/06/2013 22:43:09, no response to getwork, using as stratum
2013-06-06 22:43:10: Listener for "Default": api2.bitcoin.cz:8332 06/06/2013 22:43:10,  No JSON object could be decoded
2013-06-06 22:43:20: Listener for "Default": api2.bitcoin.cz:8332 06/06/2013 22:43:20, Failed to subscribe
2013-06-06 22:43:22: Listener for "Default": api2.bitcoin.cz:8332 06/06/2013 22:43:22, IO errors - 1, tolerance 2
2013-06-06 22:43:32: Listener for "Default" shutting down

Now the interesting part. I have since downloaded several other bitcoin miners including bitmoose which has three bitcoin miner programs included in itself. None of them were able to connect to my slush's pool. 

I've even deleted my miners account on slush's pool and that did not help. 

any ideas appreciated.

**Comment by rassalas** [comment]
The api. Servers stopped working. Go to the slush webpage and check the latest info.   It starts something like Stratum....  You may need to reconfigure your miner server and login info.

**Comment by t0mmyBerg** [comment]
I eventually did get back in using stratum.bitcoin.cz or whatever it says on slushs setup page.  using port 3333.  Doesnt give the nice immediate feedback but evetually it picks up shares and staRTS WORKING.  pardon caps lock

**Comment by candysux** [comment]
OMG THANKYOU SOOOOOO MUCH I even system restored, I tried to delete the .ini file, I even redownloaded with no results still no json error so I did what you said I picked other in guiminer put in the stratum address and 3333 and then did my slushes uid and pass and bamn I'm mining again.... HEY Kiv looks like guiminer needs another update with the correct info added for the changes that slushes pool made.... YOU ALSO NEED TO POST THIS FIX SO THAT IT IS EASIER TO FIND.... I HUNTED FOR HOURS JUST TO GET TO HERE!!!!!

TY AGAIN SO MUCH AGAIN t0mmyBerg YOU ARE MY HERO!!!!!!!!!!!!!!!!!!!!!!!!

VISIT MY SITE AND DONATE SOME BITCOIN HTTP://WWW.GTAFORLIFE.COM THE BEST GRAND THEFT AUTO MODS ON PLANET EARTH ALL MADE BY ME SWITCH DESIGNS!!!!

**Comment by rassalas** [comment]
Thank you, you are welcome.  You can donate bitcoins if this was helpful to: 1HpA1dmCjKMTM1tjzsUZ4mLw1kQ4EZFDrq
